cluster/juju/charms/trusty/kubernetes-master/hooks/__init__.py

```diff
@@ -12,5 +12,3 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
```

Comment:
These lines should not be removed (`hack/verify-boilerplate.sh` complains for this file).